### ISSUE TITLE
implementation error

### ISSUE THREAD

**Issue by daysofchasingdream** [ProfiterolePuff/DL_Stock_Prediction#1]
Hello, excuse me.
I found in your first file that this actually doesn't run, and I found a couple of obvious problems, for example the middle level of gat should not be set to 0, including in the forward, the dimension of x should be in a single column as well.

**Comment by ProfiterolePuff** [comment]
Hi, thanks for posting this issue. 

Initially, the code could be run without any error, but now it cannot.
**Changing the hyperparameter F = ["Close"] in the third chunk of code can solve the error.**
However, the model performance is dramatically reduced if only one feature can be passed into the model.

The model is originally designed with multiple features. 
I don't know why it fails now with multiple features. 
It is probably due to the new updates of the pytorch library itself.

I am aware that this implementation has some problems, as a year ago when we were able to import multiple features into this model, the MSE of this model was even worse than the baseline (simple lstm)
But the direction accuracy prediction of this hybrid model is much higher than simple lstm.

I will see whether I can solve the problem and further change the implementation of the model to increase it's performance if I have spare time.

**Comment by JonathanHuangC** [comment]
Hello, excuse me.
Have you updated the code for multiple features?
This is really helpful to me.
Thank you so much.
